spec/system/admin/item_price_policies_controller_spec.rb

```diff
@@ -30,9 +30,9 @@
 
     click_button "Add Pricing Rules"
 
-    expect(page).to have_content(table_row("#{base_price_group.name} (Internal)", "$100.00", "$0.00", "$100.00"))
-    expect(page).to have_content(table_row("#{cancer_center.name} (Internal)", "$100.00", "$25.25", "$74.75"))
-    expect(page).to have_content(table_row("#{external_price_group.name} (External)", "$125.15", "$0.00", "$125.15"))
+    expect(page).to have_content(table_row("#{base_price_group.name} (Internal)", "$100.00", "$0.00\n$100.00"))
+    expect(page).to have_content(table_row("#{cancer_center.name} (Internal)", "$100.00", "$25.25\n$100.00"))
```

Comment:
Is this right? I think the math is off now (rate is $100 and subsidy $25.25, so the final rate should be $74.75)